doc/release-notes.md

```diff
@@ -35,65 +28,109 @@ reindex or re-sync the whole chain.
+- `getblockchaininfo` now returns a new `time` field, that provides the chain tip time.
+- `fundrawtransaction` and `walletcreatefundedpsbt` when used with the `lockUnspents` argument now lock manually
+selected coins, in addition to automatically selected coins. Note that locked coins are never used in automatic coin
+selection, but can still be manually selected.
+- The `testmempoolaccept` RPC returns `vsize` and a `fee` object with the `base` fee if the transaction passes
+validation. The `testmempoolaccept` RPC now accepts multiple transactions (still experimental at the moment, API may be
+unstable).
+
+## Added RPCs
+
+- `submitchainlock` RPC allows the submission of a ChainLock signature. Note: This RPC is whitelisted for the Platform
+RPC user.
+- `getassetunlockstatuses` RPC allows fetching of Asset Unlock txs by their withdrawal index. The RPC accepts an array
+of indexes and an optional block height. The possible outcomes per each index are: `chainlocked`, `mined`, `mempooled`,
+`unknown`. Note: If a specific block height is passed on request, then only `chainlocked` and `unknown` outcomes are
+possible. This RPC is whited for the Platform RPC user.
+- `quorum dkginfo` RPC returns information about DKGs: `active_dkgs` and `next_dkg`.
+
+## Docker Build System
+
+Modified the Dockerfile to exclude the `dash-qt` graphical wallet interface from the set of binaries copied to
+`/usr/local/bin` during the Docker image build process. This change streamlines the Docker image, making it more
+suitable for server environments and headless applications, where the graphical user interface of `dash-qt` is not
+required. The update enhances the Docker image's efficiency by reducing its size and minimizing unnecessary components.
 
-The miner has been disabled for Windows binaries built via Guix. This resolves
-an issue that was preventing users from installing v20.0.3 due to false
-positive alerts from Microsoft Defender and other antivirus providers.
+## Command-line options
 
-Use a self-signed Windows code signing certificate instead of expired one. The
-expired certificate being used previously caused most antivirus software to trigger.
-The new self-signed certificate results in a greatly improved installation experience.
+The same ZeroMQ notification (e.g. `-zmqpubhashtx=address`) can now be specified multiple times to publish the same
+notification to different ZeroMQ sockets.
 
-Updated expired gpg keys for builders. Note that the actual keys themselves are the same.
-The updated keys changed their expiration dates and added additional information such as
-new email addresses.
+## Notification changes
 
-## Governance
+- `-walletnotify` notifications are now sent for wallet transactions that are removed from the mempool because they
+conflict with a new block. These notifications were sent previously before the v18.1 release, but had been broken since
+that release.
```

Comment:
```suggestion
- `-walletnotify` notifications are now sent for wallet transactions that are removed from the mempool because they
conflict with a new block. These notifications were sent previously, but have been broken since
the v18.1 release.
```